### PR TITLE
Mention the R package and other README edits

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

**Comment by maelle** [comment]
The goal is to clarify the README, that will be linked from docs.r-hub.io

File: README.md [hunk 1]
```diff
@@ -11,56 +11,19 @@ The output is JSON. The API was inspired by the
...
 
-> `http://cranlogs.r-pkg.org/badges/last-day/Rcpp`
->
-> ![](http://cranlogs.r-pkg.org/badges/last-day/Rcpp)
+* [the available badges](#badges)
 
-Total number of downloads since October of 2012, the month the
-[RStudio CRAN mirror](http://cran-logs.rstudio.com/) started publishing
-logs, with `grand-total`:
+## web API docs
```
Comment (gaborcsardi) [inline README.md]: Capitalization.

File: README.md [hunk 2]
```diff
@@ -235,3 +198,58 @@ Last day, separately for R versions and operating systems:
 Total number of downloads last week:
 
 > [`http://cranlogs.r-pkg.org/downloads/total/last-week/R`](http://cranlogs.r-pkg.org/downloads/total/last-week/R)
+
+
+## Badges 
+
+General badge Markdown syntax:
```
Comment (gaborcsardi) [inline README.md]: I don't think this is markdown syntax.

Comment (maelle) [reply README.md]: how would you call this? just badge syntax?

Comment (maelle) [reply README.md]: I made it "Endpoint for badges", is that ok?

Comment (gaborcsardi) [reply README.md]: I think you can just remove that line completely.

Comment (maelle) [reply README.md]: ok, done!

**Comment by gaborcsardi** [comment]
Can you pls squash the commits?

**Comment by maelle** [comment]
can't we squash them when merging?

**Comment by gaborcsardi** [comment]
That will create a commit with my email address, it seems.

**Comment by gaborcsardi** [comment]
So if _you_ merge that's probably also good.

**Comment by maelle** [comment]
and if _I_ squash and merge it? :-)

**Comment by maelle** [comment]
commits squashed!

**Comment by maelle** [comment]
I had missed your answer :woman_facepalming:

**Pull request merged**